spatialmath/mesh.go

```diff
@@ -242,14 +245,8 @@ func (m *Mesh) boxIntersectsVertex(b *box) bool {
 
 func (m *Mesh) distanceFromSphere(pt r3.Vector, radius float64) float64 {
```

Comment:
do we have tests for any of these function?
if not I think now would be a good time to add them.
we are going to rely heavily on this for sanding and imo it is good practice to make sure that expectations match reality.

Reply:
ugh yes we really need to have tests for this.  But this PR is really big as it is and I want to wrap it up soon so that I can move on to other work.  Would you accept a smattering of tests and a ticket to write more in the future? Getting coverage of all the functions is going to take longer than I want to spend on this

Reply:
> Would you accept a smattering of tests and a ticket to write more in the future?

sgtm, please link the ticket here it doesn't seem to exist yet.